### PR TITLE
Migrated some more test cases from ami-val.

### PR DESCRIPTION
**Pull request by narmaku**
Also fixed a false positive in cpu flags test case.

### CODE REVIEW

File: test_suite/generic/test_generic.py [hunk 1]
```diff
@@ -14,6 +14,14 @@ def test_bash_history_is_empty(self, host):
                 file_content_length = len(bash_history_file.content_string)
                 assert file_content_length == 0, f'{file_path} must be empty or nonexistent'
 
+    @pytest.mark.run_on(['rhel'])
```
Comment (jabia99) [inline test_suite/generic/test_generic.py:17]: I would add fedora to the test. And also investigate centos

**Review by jabia99 — COMMENTED** [review]
Just a small comment

File: test_suite/generic/test_generic.py (hunk 1 above)
Comment (narmaku) [reply test_suite/generic/test_generic.py:19]: I added a TODO for the future, since it's not in the priorities now.

**Review by jabia99 — APPROVED** [review]
LGTM!

**Pull request merged**